phpunit.xml.dist

```diff
@@ -0,0 +1,18 @@
+<phpunit printerClass="ScriptFUSION\PHPUnitImmediateExceptionPrinter\ImmediateExceptionPrinter"
+         bootstrap="vendor/silverstripe/framework/tests/bootstrap.php"
+         colors="true">
+
+<testsuite name="Default">
+        <directory>tests</directory>
+    </testsuite>
+
+    <listeners>
+        <listener class="\SilverStripe\Dev\TestListener" file="vendor/silverstripe/framework/src/Dev/TestListener.php"/>
```

Comment:
We don't need this in SS4 any more

Reply:
Yep, old copy-paste stuff I didn't pay attention to